### DISCUSSION TITLE
[draft-v1] Section 3.4: Here be DRAGONS

### DISCUSSION THREAD

**Discussion by pllim** [astropy/astropy-v5.0-paper#95] [Text comments] [answered]
> In addition to the two grants described above, the Project benefited from a Gemini Observatory contract under its Science User Support Department, which awarded funds for development work that supports both the Astropy and DRAGONS projects.

Do we need a footnote or citation for DRAGONS? I have no idea what it stands for although I have heard about it before. I suspect most readers would ask, "What is DRAGONS?" ❓

**Comment by adrn** [comment, marked as answer]
I addressed this in a commit that I will push soon. Thanks!
